src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -18,7 +18,7 @@ import { withRouter } from 'react-router-dom'
 import { RouteComponentProps } from 'react-router'
 import Component from './EntityCreatorComponent'
 
-const prebuiltPrefix = 'builtin-'
+const reservedNamePrefix = 'builtin-'
```

Comment:
I think it should remain as `prebuiltPrefix`. I'm not sure why github won't allow me to commnt on the line but see here:
https://github.com/Microsoft/ConversationLearner-UI/pull/982/files#diff-ecfa596a555464dcfc9af80d0fc3be98L62

I believe I said in a previous review, the constant should also be used in the `getPrebuiltEntityName` function to ensure these two stay in sync. 

Also when checking the error we use this check if the value starts with the prebuiltPrefix only when the input is not a prebuilt as these two are paired together.  A reserved name could be reserved for any reason and doesn't tell why.

Reply:
Synced both blocks of code to use the constant